messaging.rst

```diff
@@ -235,7 +137,7 @@ Fields
 Secret Reveal
 -------------
 
-Message used by the nodes to inform others that the :term:`secret` is known. Used to request an updated :term:`balance proof` with the :term:`transferred amount` increased and the lock removed.
+Message used by the nodes to inform others that the :term:`secret` is known. Used to request data to update the :term:`balance proof` with the :term:`transferred amount` increased and the lock removed.
```

Comment:
In my opinion, the initial version is better.

The node does request `an updated balance proof`. Balance proof is a concept and it is represented by https://github.com/raiden-network/raiden/blob/master/raiden/transfer/state.py#L459

At least, this is the terminology we have been using until now.

Reply:
"Balance proof" is introduced to the specs as the name of a data structure. In that sense, the node does not request a balance proof, but the necessary data to construct the updated balance proof. As I understand issue #85, it is about being accurate about this? (Although we could change the formulation here to point out the balance proof still comes from the partner, something like: "Used to request the missing data to construct the partner's updated balance proof")

Reply:
I don't think your version is incorrect. I personally preferred to maintain the balance proof concept because I think this is more important than the actual implementation (transferred amount, locked, locksroot). In the sense that the balance proof can (maybe) be implemented in other ways by people, but serves the same scope.

This being said, maybe some input from @LefterisJP or @hackaugusto might be better regarding this.